doc/mesh_filter/mesh_filter_tutorial.rst

```diff
@@ -0,0 +1,111 @@
+Mesh Filter with UR5 and Kinect
+===============================
+
+MoveIt's mesh filter functionality removes your robot's geometry from its point cloud! If your robot's arm is in your depth sensor's view, the points associated with the arm are subtracted from the point cloud. 
+
+This is accomplished by giving the original point cloud, the robot's transforms (tf) and the robot's URDF as inputs.
```

Comment:
The "(tf)" here refers to the topic `/tf`, right? Maybe use the leading slash and fixed-width font to make that clear: ``` ``\tf`` ```